schematools/importer/ndjson.py

```diff
@@ -8,11 +8,12 @@
 class NDJSONImporter(BaseImporter):
     """Import an NDJSON file into the database."""
 
-    def parse_records(self, file_name, dataset_table, **kwargs):
+    def parse_records(self, file_name, dataset_table, db_table_name, **kwargs):
```

Comment:
Ha Jan, vraag, waarom heb je bij db_table_name geen =None gebruikt als default? Waarschijnlijk omdat je verplicht bent om deze parameter atlijd te vullen met None / werkelijke waarde?

Reply:
Goeie Chris. Hij moet wel degelijk de default None hebben, zo is het namelijk ook in de superclass gedefineerd, ik pas het aan.